go/web.config

```
@@ -10,6 +10,12 @@
                   <action type="Redirect" url="https://marc.durdin.net/2018/03/the-case-for-keyman/" />
                 </rule>
 
+                <!-- Developer 10.0 onward redirects for package guide -->
+                <rule name="developer-help-mobile / developer-help-packages" stopProcessing="true">
+                    <match url="^([1-9][0-9])([0-9])/developer-help-(mobile|packages)(/)?$" />
+                    <action type="Redirect" url="https://help.keyman.com/developer/{R:1}.0/guides/distribute/packages" />
```

```suggestion
                    <action type="Redirect" url="https://help.keyman.com/developer/{R:1}/guides/distribute/packages" />
```

This goes with that ... above